processing-service-core/tenant-config-options.md

```diff
@@ -0,0 +1,200 @@
+# Overview of tenant specific config options
+
+## Setting tenant configurations
+
+A tenant can specify custom values for configuration that can then be used when creating their workflow for the enrichment process.
+A tenant can only specify a custom configuration value for a configuration that already has a global configuration option, if a global configuration option does not exist for the configuration that the tenant tries to set the service will return an APIException to make the user aware that a global configuration must be created for first.
+If a tenant does not have a custom configuration set for the key provided in this call then one will be created for the tenant, if one does exist then its value will be updated with the value provided.
```

Comment:
"that a global configuration must be created for first." should be "must be created first" or "must be created for the tenant configuration".